warhorn/warhorn.py

```diff
@@ -1198,12 +1181,25 @@ def assemble_warrior():
 
     check_basic_requirements(logfile, config_file_name, console_log_name,
                              print_log_name, python_executable)
-    get_dependencies(logfile, print_log_name, config_file_name)
+    node = get_node(config_file_name, 'virtualenv')
+    if node is not False and get_attribute_value(node, 'activate') == 'yes':
+        if get_attribute_value(node, 'install') == 'yes':
+            install_depen('virtualenv==15.1.0', 'virtualenv', logfile, print_log_name)
+        war_tag = get_node(config_file_name, 'warriorframework')
+        if war_tag is False:
+            print_error("warriorframework is a mandatory repo. Please add and rerun",
+                        logfile, print_log_name)
+            setDone(1)
+            getDone(logfile, print_log_name)
+        dest = get_attribute_value(war_tag, 'destination')
+        virtualenv_activated = activate_virtualenv(node, dest, logfile, print_log_name)
```

Comment:
If activate is set to yes for virtualenv, I think it is reasonable to keep virtualenv activated after installation finish, that way user don't need to activate virualenv again before running warrior

Reply:
Yes, we can not achieve the ve prompt by exiting the warhorn script. Please approve or let me know, what you are expecting.

Reply:
Hi @navvrnd 
I discussed with Brian about the activate attribute in the virtualenv tag
we agreed that the activate attribute isn't really necessary here if it only indicates whether to install dependencies inside the virtualenv or not
If the user gives a name for the virtualenv, then we should use the virtualenv
if the name field is blank, then we shouldn't use it
there's no need for an activate attribute
Can you please make the change for this?

Reply:
Yes, done the changes. Please review and approve.

Reply:
@kaheichan Can you please review this?